admin/templates/configuration/etc/log4cxx.czar.properties

```diff
@@ -3,7 +3,7 @@
 # used by Qserv-czar
 #
 
-log4j.rootLogger=DEBUG, FILE
+log4j.rootLogger=DEBUG, CONSOLE
```

Comment:
I want to reduce the number of log files produced by mysql-proxy, easiest way to do it now is to log everything to standard output (console) and redirect it to a log file (mysql-proxy-lua.log).